src/event/metric.rs

```diff
@@ -233,7 +233,7 @@ impl Display for Metric {
     /// and `Y` is the value or bucket.
     ///
     /// example:
-    /// ```
+    /// ```ignore
```

Comment:
This seems fine, but it also looks like we can use ```text to tell Rust that it is just text.

https://doc.rust-lang.org/rustdoc/documentation-tests.html#attributes

Reply:
Good note, I'll change!